src/OmniSharp.MSBuild/ProjectFile/PropertyConverter.cs

```diff
@@ -165,10 +165,9 @@ public static OutputKind ToOutputKind(string propertyValue)
             }
         }
 
-        public static NuGetVersion ToNuGetVersion(string propertyValue)
+        public static VersionRange ToVersionRange(string propertyValue)
         {
-            NuGetVersion version;
-            if (NuGetVersion.TryParse(propertyValue.Trim(), out version))
+            if (VersionRange.TryParse(propertyValue.Trim(), out var version))
```

Comment:
There's a bunch of them: `NuGetVersion`, `SemanticVersion`, `VersionRange`, `FloatRange`. They're all in NuGet.Versioning. `NuGetVersion` is a specific version, used for package identity. A range wouldn't be suitable for that.